bounties/npm/ts-dot-prop/1/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": 1,
+    "DisclosureDate": "2020-09-13",
+    "AffectedVersionRange": "*",
+    "Summary": "Prototype Pollution",
+    "Contributor": {
+        "Discloser": "Chandra Srikanth Prathi",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "ts-dot-prop",
+        "URL": "https://www.npmjs.com/package/ts-dot-prop",
+        "Downloads": "4000/month"
+    },
+    "CWEs": [
+        {
+            "ID": "CWE-74",
+            "Description": ""
+        }
+    ],
+    "CVSS":
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "L",
+        "PR": "N",
+        "UI": "N",
+        "S": "U",
+        "C": "H",
+        "I": "H",
+        "A": "H",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "9.8"
```

Comment:
Too high in my opinion

Reply:
Thank you for the suggestion @Mik317. Modified the CVSS score.